File: 0674-longest-continuous-increasing-subsequence/0674-longest-continuous-increasing-subsequence.cpp
class Solution {
public:
    int findLengthOfLCIS(vector<int>& nums) {
        int low =0;
        int high=1;
        int n=nums.size();
        int max_ht=0;
        while(low<n && high<n){
            if(nums[high-1]<nums[high]){
                high++;
            }
            else{
                max_ht=max(max_ht,high-low);
                low=high;
                high=low+1;
            }
        }
        max_ht=max(max_ht,high-low);
        return max_ht;
    }
};

File: 0674-longest-continuous-increasing-subsequence/0674-longest-continuous-increasing-subsequence.py
class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        low=0
        high=1
        max_ht=0
        n=len(nums)
        while low<n and high<n :
            if nums[high-1]<nums[high]:
                high+=1
            else :
                max_ht=max(max_ht,high-low)
                low=high
                high=low+1
        max_ht=max(max_ht,high-low)
        return max_ht